### ISSUE TITLE
Bug: google drive don't appear on cloud

### ISSUE THREAD

**Issue by gilnett** [files-community/Files#16470]
### Description

https://cdn.discordapp.com/attachments/1305952471541350500/1305952471817916426/image.png?ex=67378984&is=67363804&hm=04322b068b2a70ba437f8c30706be15a14f4cbaf9202d8cf5d5d2c7dabaf84c3&
The Google Drive app isn't showing up in the right place. I've restarted "Files" and Windows 11, but it hasn't changed anything. I saw that this bug was supposed to be patched, but I'm still experiencing it, so I'm asking if there are other ways to fix it.

### Steps To Reproduce

1.download google drive
2.start google drive
3.open files 
4.bug start

### Files Version

3.7.12.0

### Windows Version

23h2

### User ID

f16d649c-d01a-4012-a5bd-85efa026a657"

### Log File

[debug.log](https://github.com/user-attachments/files/17758243/debug.log)

**Comment by yaira2** [comment]
I believe there is a setting in Google Drive that needs to be enabled. @Josh65-2201 might know more about this.

**Comment by Josh65-2201** [comment]
There is a support post on Discord, It doesn't how in cloud drive with any of the options in Google drive and a Files restart.

**Comment by kawadumax** [comment]
<img height="512" alt="Image" src="https://github.com/user-attachments/assets/d25967d3-c4bd-4b39-8b03-329d639d9548" />

win version: 10.0.26200.7623
App version: 4.0.24.0

Is this issue still being tracked in 2026?

I'm encountering the same behavior on Windows 11. I installed Google Drive via winget and have it set to "Stream files" mode.

Currently, it is recognized as a mounted drive (G:) and does not appear in the "Cloud" storage section of the Files app. While this may be the intended behavior of Google Drive's virtual drive system, it would be much more consistent and intuitive if it were integrated into the Cloud section, similar to how OneDrive is displayed.

2026年現在もこの問題は継続中でしょうか。 Windows 11環境にて、wingetでインストールしたGoogle Drive（ストリーミング設定）を使用していますが、クラウドセクションではなく「Gドライブ」としてマウントされた状態で表示されます。Google Drive側の仕様という側面もあるかと思いますが、OneDriveのように「クラウド」セクションに統合されると、より一貫性があって使いやすくなると感じています。

**Comment by Josh65-2201** [comment]
Yes, this issue will be closed as complete when fixed. Google Drive is unfortunately hard to detect correctly with the way they make the client work
